### PR TITLE
add forceLoadFixtures functional into 2.4

### PR DESCRIPTION
**Pull request by bscheshirwork**
migrate from `add forceLoadFixtures functional` #4748

In some case we need **insert data from fixture to database** Independence of check empty($this->loadedFixtures).
For example for second test with same fixtures. + change database from another Codeception module (`Db` cleaning)
without this PR option the db data will be lost but fixtures data still alive

>so... I strongly against to idea "make this method is public" now (after 2.4 changes)
And yii2 module now have many news options. We can add another one without any problem in design...
https://github.com/Codeception/Codeception/pull/4748#issuecomment-380396405

example
```yml
suite_namespace: api\tests\functional
actor: FunctionalTester
modules:
    enabled:
        - Asserts
        - \api\tests\Helper\Functional
        # add a framework module here
        - \bscheshirwork\Codeception\Module\DbYii2Config: # extends \Codeception\Module\Db
            dump: ../common/tests/_data/dump.sql # relative path from "codeception.yml"
            populate: true
            cleanup: true
        - REST:
            url: /v1/
            depends: Yii2 # module with depends of Yii2 will be set after module DbYii2Config
        - Yii2:
            cleanup: true # remove fixture in Module\Yii2::_after and load fixture in  Module\Yii2::_before. In this case data from fixtures insert to db AFTER DbYii2Config populate
            #transaction: false # true = SQLSTATE[42000]: Syntax error or access violation: 1305 SAVEPOINT LEVEL1 does not exist
            transaction: true # enable directly
            forceLoadFixtures: true # in second test data force reload into database.
```

### CODE REVIEW

**Comment by bscheshirwork** [comment]
@samdark @SamMousa @DavertMik how about this?

**Comment by SamMousa** [comment]
I'm not against adding more options in general.
I don't understand the use for this option.
If you have cleanup enabled in Yii module, as in your example, `loadedFixtures` is reset to `[]`. So in the second test it should already be reloaded.

File: src/Codeception/Module/Yii2.php [hunk 1]
```diff
@@ -322,7 +326,7 @@ private function loadFixtures($test)
             $this->debugSection('Fixtures', 'Opened database connection: ' . $event->sender->dsn);
             $connections[] = $event->sender;
         });
-        if (empty($this->loadedFixtures)
+        if (($force || empty($this->loadedFixtures))
```
Comment (SamMousa) [inline src/Codeception/Module/Yii2.php]: Why not just check config directly? 
`if ($this->_getConfig('forceLoadFixtures') && ...`

That way you don't need to change the function and its calls.

**Comment by bscheshirwork** [comment]
> If you have cleanup enabled in Yii module, as in your example, loadedFixtures is reset to []. So in the second test it should already be reloaded.

so... I migrate to 2.4 and I also can't need it now. 
`loadedFixtures` is reset to `[]` in common case...

**Pull request closed without merging**